src/components/arbeidsoppgaver/ArbeidsoppgaveCard.tsx

```diff
@@ -25,20 +25,36 @@ import { Arbeidsoppgave } from "../../types/oppfolgingsplan";
 
 interface Props {
   innloggetFnr: string;
+  arbeidstakerFnr?: string;
   arbeidsoppgave: Arbeidsoppgave;
   readonly?: boolean;
 }
 
 export const ArbeidsoppgaveCard = ({
   innloggetFnr,
+  arbeidstakerFnr,
   arbeidsoppgave,
   readonly = true,
 }: Props) => {
   const { isAudienceSykmeldt } = useAudience();
   const type = arbeidsoppgave.gjennomfoering?.kanGjennomfoeres;
   const [editererArbeidsoppgave, setEditererArbeidsoppgave] = useState(false);
   const aktoerHarOpprettetElement =
-    innloggetFnr === arbeidsoppgave.opprettetAv.fnr;
+    innloggetFnr === arbeidsoppgave.opprettetAv.fnr ||
+    (innloggetFnr !== arbeidstakerFnr &&
+      arbeidstakerFnr !== arbeidsoppgave.opprettetAv.fnr);
+
+  const opprettetAvText = () => {
```

Comment:
Mulig denne også burde ha en slags util metode? Jeg har brukt stor "A" på arbeidstaker/arbeidsgiver. https://github.com/navikt/oppfolgingsplan-frontend/blob/f799717d6a817ad90a199608a635d646738c2854/src/components/blocks/dialog/Dialog.tsx#L48

Reply:
Enig, men vil gjerne egen pr for det